modules/ROOT/pages/amazon/amazon-dynamodb-connector-studio.adoc

```diff
@@ -0,0 +1,153 @@
+
+In Studio, create a new Mule project in which to add and configure the connector:
+
+. In Anypoint Studio, from the main menu, select *File > New > Mule Project*.
+. Enter a name for your Mule project and click *Finish*.
+
+[[add-connector-to-project]]
+== Add the Connector to Your Mule Project
+
+Add Amazon DynamoDB Connector to a Mule project to automatically populate the XML code with the connector's namespace and schema location, as well as to add the required dependencies to the project's `pom.xml` file:
+
+. In the *Mule Palette* view, click *(X) Search in Exchange*.
+. In *Add Modules to Project*, type `amazon dynamodb` in the search field.
+. Click *Amazon DynamoDB* in *Available modules*.
+. Click *Add*.
+. Click *Finish*.
+
+Adding a connector to a Mule project in Studio does not make that connector available to other projects in your Studio workspace.
+
+[[configure-input-source]]
+== Configure an Input Source
+
+An input source initiates a flow when a specified condition is met. You can configure one of these input sources to use with Amazon DynamoDB Connector:
+
+* `HTTP Listener`, which initiates a flow each time it receives a request on the configured host and port
+* `Scheduler`, which initiates a flow when a time-based condition is met
+
+For example, to configure `HTTP Listener`, follow these steps:
+
+. In the *Mule Palette* view, select *HTTP* > *Listener*.
+. Drag *Listener* to the Studio canvas.
+. On the *Listener* configuration screen, optionally change the value of the *Display Name* field.
+. Specify a value for the *Path* field.
+. Click the plus sign (+) next to the *Connector configuration* field to configure a global element that can be used by all instances of `HTTP Listener` in the app.
+. On the *General* tab, specify connection information.
+. On the *TLS* tab, optionally specify TLS information.
+. On the *Advanced* tab, optionally specify reconnection information, including a reconnection strategy.
+. Click *Test Connection* to confirm that Mule can connect with the specific server.
+. Click *OK*.
+
+[[add-connector-operation]]
+== Add a Connector Operation to the Flow
+
+When you add a connector operation to your flow, you immediately define a specific operation for that connector to perform.
+
+To add an operation for Amazon DynamoDB Connector, follow these steps:
+
+. In the *Mule Palette* view, select *Amazon DynamoDB* and then select the desired operation.
+. Drag the operation onto the Studio canvas to the right of the input source.
+
+[[configure-global-element]]
+== Configure a Global Element for the Connector
+
+When you configure a connector, it is a good practice to configure a global element that can be used by all instances of that connector in the app. Configuring a global element requires you to provide the authentication credentials the connector uses to access the target Amazon DynamoDB system. Amazon DynamoDB Connector supports both Basic authentication and Role authentication.
```

Comment:
Consideration:
I've been gathering info about the practice of (not) using "MuleSoft recommends"; alas, I lost my Notes in my laptop swap. However, I remember that Salesforce uses "it is best to," while I have been recommending instead that we simply document the best way to do something:
```suggestion
When you configure a connector, configure a global element that can be used by all instances of that connector in the app. Configuring a global element requires you to provide the authentication credentials the connector uses to access the target Amazon DynamoDB system. Amazon DynamoDB Connector supports both Basic authentication and Role authentication.
```

Reply:
Changing to "it is best to..."